src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -58,6 +58,21 @@ import ActionsPaneFooter from './ActionsPaneFooter';
 
 const {TabPane} = Tabs;
 
+const formSupportedResourceKinds = [
```

Comment:
we should move this to the ResourceKindHandler - i.e. add properties for formSchema and formUiSchema to it and set those correspondingly - the ActionsPane and Form component can get the ResourceKindHandler for the selected resource and discover if these are available